tests/tests_auth/test_controllers.py

```diff
@@ -0,0 +1,31 @@
+
+from flask import url_for
+from tests.main import BaseTestCase
+
+
+class TestLogIn(BaseTestCase):
+
+    def test_if_login_form_renders(self):
+        response = self.app.test_client().get(url_for('auth.login'))
+        self.assertEqual(response.status_code, 200)
+        self.assert_template_used('auth/login.html')
+
+    def test_login(self):
+
+
+
+class TestLogOut(BaseTestCase):
```

Comment:
Did you actually run these tests? There is no way that this test currently can pass.

Reply:
Working on it and getting it fixed.